test/components/generators/chat/conftest.py

```diff
@@ -1,14 +0,0 @@
-# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
-#
-# SPDX-License-Identifier: Apache-2.0
-import pytest
-
-from haystack.dataclasses import ChatMessage
-
-
-@pytest.fixture
-def chat_messages():
-    return [
-        ChatMessage.from_system("You are a helpful assistant speaking A2 level of English"),
-        ChatMessage.from_user("Tell me about Berlin"),
-    ]
```

Comment:
we were using this conftest on test/components/generators/chat/test_hugging_face_local.py only
I removed it because it added indirection